src/test/bin/quic_gtest.h

```diff
@@ -692,3 +692,76 @@ std::ostream& operator << (std::ostream& o, const RebindPaddingArgs& args) {
 class WithRebindPaddingArgs : public testing::Test,
     public testing::WithParamInterface<RebindPaddingArgs> {
 };
+
+struct TlsConfigArgs {
+
+    QUIC_CREDENTIAL_TYPE CredType;
+    CXPLAT_TEST_CERT_TYPE CertType;
+
+    static ::std::vector<TlsConfigArgs> Generate() {
+        ::std::vector<TlsConfigArgs> List;
+        for (auto CredType : {
+#ifdef _WIN32
+            QUIC_CREDENTIAL_TYPE_CERTIFICATE_HASH,
+            QUIC_CREDENTIAL_TYPE_CERTIFICATE_HASH_STORE,
+            QUIC_CREDENTIAL_TYPE_CERTIFICATE_CONTEXT,
+#else
+            QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE,
+            QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE_PROTECTED,
+            QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12
```

Comment:
We don't support any of these on Windows?

Reply:
The code path in `CxPlatTlsSecConfigCreate` is no different on Windows or POSIX if those credential types are presented.